effects.d.ts

```diff
@@ -3,15 +3,15 @@
  * There is no js module backing this up.
  */
 import { Action } from 'redux';
-import { Task, Channel, Buffer } from 'redux-saga';
+import { Task, TakeableChannel, FlushableChannel Buffer } from 'redux-saga';
```

Comment:
```suggestion
import { Task, TakeableChannel, FlushableChannel, Buffer } from 'redux-saga';
```

Reply:
whoops. Thanks.

Reply:
Welcome. Just passing by and saw a missing comma 😄